modules/5_segmentation/include/segmentation.hpp

```diff
@@ -2,9 +2,12 @@
 #include <opencv2/opencv.hpp>
 #include <inference_engine.hpp>
 
+using namespace InferenceEngine;
+using namespace cv;
+
 // Compute Dice score over to binary masks
 float Dice(const cv::Mat& a, const cv::Mat& b);
-
+Blob::Ptr wrapMatToBlob(const Mat& m);
```

Comment:
Please remove this definition from a header file

Reply:
Done

Reply:
Please push the changes to GitHub